recipes/scikit-cycling/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "scikit-cycling" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "3acaed9ccfdfa34f43e75342bd86c18bb3ad4b940a24eedb78f674364fba1076" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-cycling/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export CC=clang  # [osx]
+    - export CXX=clang++  # [osx]
+    - python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+    - cython
+    - numpy 1.9.*  # [not (win and py36)]
+    - numpy 1.11.*  # [win and py36]
```

Comment:
In practice, some people use `numpy 1.8.*` as the lower limit. Not required though.

ref: https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
I think it was the reason to pin at 1.9:
https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.22442/job/wa6vfqfdqdq3n0bp

Reply:
The link from @jakirkham's link says:
```
  - numpy 1.8.*  # [not (win and (py35 or py36))]
  - numpy 1.9.*  # [win and py35]
  - numpy 1.11.*  # [win and py36]
```

Reply:
yep I should have read that sorry